tests/entrypoints/test_openai_server.py

```diff
@@ -687,6 +687,22 @@ async def test_response_format_json_object(server, client: openai.AsyncOpenAI):
     assert loaded == {"result": 2}, loaded
 
 
+async def test_complex_message_content(server, client: openai.AsyncOpenAI):
+    resp = await client.chat.completions.create(model=MODEL_NAME,
```

Comment:
let's set temperature to 0 and seed so the output is determinstically 2

Reply:
Good suggestion, done!